### PR TITLE
Fix invoices not showing

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

**Comment by hedudelgado** [comment]
Oh my god @alexnoox , shame on me, sorry! I deleted the locales. The Pr is clean now.

**Comment by hedudelgado** [comment]
https://github.com/maestrano/mno-enterprise/pull/494

File: src/app/views/company/invoices/organization-invoices.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <div class="row td-row" ng-repeat="invoice in invoices">
     <div class="cell col-sm-3">
       <span class="cell-label visible-xs-inline">{{ 'mno_enterprise.templates.dashboard.organization.invoices.period' | translate }}:<br/></span>
-      {{ invoice.period }}
+      {{ invoice.started_at| amCalendar}} - {{ invoice.ended_at| amCalendar}}
```
Comment (alexnoox) [inline src/app/views/company/invoices/organization-invoices.html]: I think you should use `amDateFormat:'L'`

**Comment by hedudelgado** [comment]
Fixed @alexnoox . Thanks

**Pull request merged**